### PR TITLE
Tab/single user

### PR DESCRIPTION
**Pull request by OnizukaX**
Fixing issues: #1 and #20 

Implemented as following:
- front and rear light segments - each as (lower, upper) pairs [1; num_pixels] - are stored with Nffs as Peripheral settings (e.g. "traffic_front_lower")
- default values are provided by the settings manager
- colors are front = rgb(255, 255, 255), rear = rgb(255, 0, 0)
- settings are configurable via the Central in the Popover, inputs are ranges [1; num_pixels]
- if front/rear overlap, rear overwrites the front as per the sequential flow implemented
- the LED library directly reads the settings (pointer passed over during init.) allowing one to change settings on the fly without having to reboot the Peripheral
- a new TrafficMode has been introduced to the main tab

note: the lightninh direction is part of the issue #19

### CODE REVIEW

File: periph_sw/nRF52_RGB_Strip/eeprom_handler.h [hunk 1]
```diff
@@ -34,10 +43,14 @@ class EEPROM_Handler
     {
     }
 
-    void static load(Settings &settings)
+    void configure(void)
```
Comment (Beardmix) [inline periph_sw/nRF52_RGB_Strip/eeprom_handler.h]: This function should also be static, right?

File: periph_sw/nRF52_RGB_Strip/nRF52_RGB_Strip.ino [hunk 2]
```diff
@@ -4,14 +4,10 @@
 #include "ble_handler.h"
 #include "eeprom_handler.h"
 
-/* Need to undefine min and max in order to compile <String>. */
-#undef max
```
Comment (Beardmix) [inline periph_sw/nRF52_RGB_Strip/nRF52_RGB_Strip.ino]: Why is this not needed any more?

File: periph_sw/nRF52_RGB_Strip/led_lib.h [hunk 3]
```diff
@@ -204,14 +210,32 @@ class CtrlLED
...
     }
 
+    void modeTraffic(void)
+    {
+        this->setPixelsOff(); // Init: switch evertyhing off.
+
+        // Front.
+        for (int i = std::max(this->p_settings->traffic_front_lower - 1, 0U); i < this->p_settings->traffic_front_upper; ++i)
+        {
+            strip.setPixelColor(i, strip.Color(255, 255, 255));
```
Comment (Beardmix) [inline periph_sw/nRF52_RGB_Strip/led_lib.h]: A light yellow would be better. Pure white is generally too strong and aggressive, even at lower brightness settings.

File: periph_sw/nRF52_RGB_Strip/eeprom_handler.h (hunk 1 above)
Comment (OnizukaX) [reply periph_sw/nRF52_RGB_Strip/eeprom_handler.h]: Could be yes, will set it static.

File: periph_sw/nRF52_RGB_Strip/nRF52_RGB_Strip.ino (hunk 2 above)
Comment (OnizukaX) [reply periph_sw/nRF52_RGB_Strip/nRF52_RGB_Strip.ino]: There was a namesapce conflict between libraries. I replaced the calls by constrain() and std::max() where needed.

File: periph_sw/nRF52_RGB_Strip/led_lib.h (hunk 3 above)
Comment (OnizukaX) [reply periph_sw/nRF52_RGB_Strip/led_lib.h]: Agreed, will be changed later.

**Pull request merged**